lib/features/manage_account/data/local/notification_setting_cache_manager.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:core/utils/app_logger.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+
+class NotificationSettingCacheManager {
+  static const keyAppNotificationSettingCode = 'KEY_APP_NOTIFICATION_SETTING_CODE';
+
+  final SharedPreferences _sharedPreferences;
+
+  NotificationSettingCacheManager(this._sharedPreferences);
+
+  Future<void> persistAppNotificationSettingCache(bool isEnabled) async {
```

Comment:
```suggestion
  Future<void> persistAppNotificationSettingCache({required bool isEnabled}) async {
```
to avoid magic value

Reply:
Done